Tests/js/views/services/assets/ez-dashboardblocksviewservice-tests.js

```diff
@@ -0,0 +1,226 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-dashboardblocksviewservice-tests', function (Y) {
+    'use strict';
+
+    var successTest,
+        errorTests,
+        getViewParametersTest,
+        searchPluginRegisterTest,
+        rootLocationDefaultValueTest,
+        loadRootLocationErrorMessage = 'Cannot load root location',
+        loadLocationModelErrorMessage = 'Cannot load root location data into model';
+
+    Y.eZ.Location = Y.Model;
```

Comment:
that's a bit dirty, this should be done in each `setUp` where that's needed and the corresponding clean up should be done in the corresponding `tearDown`